<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>695</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Typo Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/icons_db.qrc">
    <normaloff>:/app.png</normaloff>:/app.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.999999000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#cccccc;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#404040;font:#fcfcfc;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblActiveFont">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fcfcfc;</string>
           </property>
           <property name="text">
            <string>Default Font</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="chkObserve">
           <property name="styleSheet">
            <string notr="true">color:#fcfcfc;</string>
           </property>
           <property name="text">
            <string>Observe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPdbExport">
           <property name="styleSheet">
            <string notr="true">color:#ff0000;</string>
           </property>
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldTrans">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>40</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblFontSize">
           <property name="styleSheet">
            <string notr="true">color:#fcfcfc;</string>
           </property>
           <property name="text">
            <string>Size: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbFontSize">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fcfcfc;</string>
           </property>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>36</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>48</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>72</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>96</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>144</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>180</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>220</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>250</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbTextSamples">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#fcfcfc;</string>
           </property>
           <item>
            <property name="text">
             <string>custom text..</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>arabic_1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>arabic_2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>kurdish</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>marks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>persian_arabic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>persian_arabic2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Quran</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <underline>false</underline>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#ffffff;</string>
      </property>
      <property name="locale">
       <locale language="Arabic" country="Iraq"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoAll</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources/icons_db.qrc">
     <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTo_PDF">
   <property name="text">
    <string>To PDF</string>
   </property>
   <property name="toolTip">
    <string>Export as PDF Document</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons_db.qrc"/>
  <include location="../resources/samples.qrc"/>
 </resources>
 <connections/>
</ui>
